src/main/kotlin/snyk/iac/IacService.kt

```diff
@@ -0,0 +1,43 @@
+package snyk.iac
+
+import com.google.gson.Gson
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.project.Project
+import io.snyk.plugin.cli.CliError
+import io.snyk.plugin.cli.ConsoleCommandRunner
+import io.snyk.plugin.services.CliService
+import snyk.common.SnykError
+
+/**
+ * Wrap work with Snyk CLI for IaC (`iac test` command).
+ */
+@Service
+class IacService(project: Project) : CliService<IacResult>(
+    project = project,
+    cliCommands = listOf("iac", "test")
+) {
+
+    override fun getErrorResult(errorMsg: String): IacResult = IacResult(null, SnykError(errorMsg, projectPath))
+
+    override fun convertRawCliStringToCliResult(rawStr: String): IacResult =
+        when {
+            rawStr == ConsoleCommandRunner.PROCESS_CANCELLED_BY_USER -> IacResult(null, null)
```

Comment:
Do we need a case if `rawStr` is empty?

Reply:
will be handled in separate PR for https://github.com/snyk/snyk-intellij-plugin/issues/153

Reply:
correct, and even more: JsonSyntaxException case as well.